manifests/crd/v1/pingcap.com_backups.yaml

```diff
@@ -19,6 +19,15 @@ spec:
   scope: Namespaced
   versions:
   - additionalPrinterColumns:
+    - description: the type of backup, such as full, db, table. Only used when Mode
+        = snapshot.
+      jsonPath: .spec.backupType
```

Comment:
what is the backuptype? i can't find this in br's doc https://docs.pingcap.com/zh/tidb/dev/backup-and-restore-overview
can you explain more

Reply:
backuptype is the backup data type, like `full`, `db`, `table`, commands in br is https://docs.pingcap.com/zh/tidb/dev/use-br-command-line-tool#%E5%91%BD%E4%BB%A4%E5%92%8C%E5%AD%90%E5%91%BD%E4%BB%A4, like `br backup full` or `br backup db`.
by the way, log backup's command is like `br log start`

Reply:
thanks